pgo/cmd/backup.go

```diff
@@ -82,6 +86,7 @@ func init() {
 	backupCmd.Flags().StringVarP(&BackupOpts, "backup-opts", "", "", "The options to pass into pgbackrest.")
 	backupCmd.Flags().StringVarP(&Selector, "selector", "s", "", "The selector to use for cluster filtering.")
 	backupCmd.Flags().StringVarP(&PVCName, "pvc-name", "", "", "The PVC name to use for the backup instead of the default.")
+	backupCmd.Flags().StringVarP(&PGDumpDB, "database", "d", "postgres", "The name of the database pgDump will backup.")
```

Comment:
Just noticing we have `pgDump` (camelCase) here but `pgdump` (all lowercase) in other command descriptions.  Would be nice to make this consistent (same applies to restore).

Reply:
sure thing, updated to 'pgdump' in both cases.